03_letters/anishaNair/main.go

```diff
@@ -0,0 +1,39 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"sort"
+)
+
+var out io.Writer = os.Stdout
+
+func main() {
+	sortAndPrint(countLetters("aba"))
+}
+
+func countLetters(s string) map[rune]int {
+	elementMap := make(map[rune]int)
+	for _, c := range s {
+		_, ok := elementMap[c]
+		if !ok {
+			elementMap[c] = 1
+		} else {
+			elementMap[c]++
+		}
+	}
+	return elementMap
+}
+
```

Comment:
The lab asks for a function `sortLetters(map[rune]int) []string` which is missing. Most of its functionality is in `sortAndPrint` though. Shouldn't be too hard to modify that function to return a sorted array.

Reply:
fixed it